source/geometries/Next100TrackingPlane.h

```diff
@@ -47,10 +47,11 @@ namespace nexus {
     void PlaceSiPMBoardColumns(G4int, G4double, G4double, G4int&, G4LogicalVolume*);
 
   private:
-    G4double gate_tp_dist_;
     const G4double copper_plate_diameter_, copper_plate_thickness_;
     const G4double distance_board_board_;
 
+    G4double gate_tp_dist_;
+
```

Comment:
Yes, the thing is that this variable needs to be set from outside the class, in the `Next100InnerElements` class. Since the Next100TrackingPlane class is only used in this way, we didn't worry about initializing the variable. If necessary, we should set it to something like `-1.e9` and put a check that the value makes sense, but I'm not sure if it's needed.